sumdbaudit/docker/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.9"
+
+# TODO(mhutchinson): This file-based sharing should be replaced with a
+# client/server DB running in its own container.
+volumes:
+  sqlite-data:
+
+services:
+  mirror:
+    build: 
+      context: ../..
+      dockerfile: ./sumdbaudit/docker/mirror/Dockerfile
+    volumes:
+      - sqlite-data:/var/cache/sumdb
+    command: [
+      "--alsologtostderr",
+      "--db=/var/cache/sumdb/mirror.db"
+    ]
+    restart: always
+  witness:
+    build: 
+      context: ../..
+      dockerfile: ./sumdbaudit/docker/witness/Dockerfile
+    volumes:
+      - sqlite-data:/var/cache/sumdb
+    command: [
+      "--alsologtostderr",
+      "--db=/var/cache/sumdb/mirror.db"
+    ]
+    ports:
+     - "8080:8000"
```

Comment:
consider setting the flag on the job to made it listen on `8000` too?
Could be a little confusing with docker doing this PAT?

Reply:
I've explicitly added the port address so that they are both listening on 8080 and the docker server is just opening the port.